core/templates/dev/head/pages/exploration_player/AnswerClassificationService.js

```diff
@@ -149,10 +126,7 @@ oppia.factory('AnswerClassificationService', [
               answerClassificationResult = (
                 AnswerClassificationResultObjectFactory.createNew(
                   answerGroups[predictedAnswerGroupIndex].outcome,
-                  predictedAnswerGroupIndex,
-                  findClassifierRuleIndex(
-                    answerGroups[predictedAnswerGroupIndex]),
-                  STATISTICAL_CLASSIFICATION));
+                  predictedAnswerGroupIndex, 0, STATISTICAL_CLASSIFICATION));
```

Comment:
Why 0? Should it be null? (Make sure to update the documentation for AnswerClassificationResult, too, and check how the index is used.)

Reply:
Actually in Default classification we use 0 as rule index and so I thought we can do same here but now I see that it is wrong as answer group index does not refer to default answer group. Will fix this in next commit

Reply:
Done